core/src/test/java/com/sequenceiq/cloudbreak/controller/validation/stack/StackRequestValidatorTest.java

```diff
@@ -101,6 +172,316 @@ public void testWithBothGroupContainsDifferentValues() {
         assertEquals(2L, validationResult.getErrors().size());
     }
 
+    @Test
+    public void testWhenBlueprintIsSharedServiceReadyAndThereIsOnlyHiveRdsWithNameThenErrorComes() {
+        when(blueprintJson.getMap()).thenReturn(Map.of(SHARED_SERVICE_READY_BP_TAG, true));
+        StackRequest request = stackRequest();
+        request.getClusterRequest().setRdsConfigNames(Set.of(TEST_HIVE_NAME));
+        request.getClusterRequest().setRdsConfigJsons(Collections.emptySet());
```

Comment:
Does cloudbreak also initializes these fields to empty sets during converting?

Reply:
yes